setupext.py

```diff
@@ -906,7 +906,7 @@ def check(self):
             try:
                 check_include_file(get_include_dirs(), 'ft2build.h', 'freetype')
             except CheckFailed:
-                check_include_file(get_include_dirs(), 'freetype2\\ft2build.h', 'freetype')
+                check_include_file(get_include_dirs(), os.path.join('freetype2', 'ft2build.h'), 'freetype')
```

Comment:
This block should only be executed if `sys.platform` is windows, but I guess the change is reasonable. Does it fix a known bug?

Reply:
No, it may also be executed IFF you don't have `ft2build.h` installed in `/usr/include`. I.e. if you install freetype in a non-default location and you want to link it using `setup.cfg` and basedirlist, then you have problems because freetype installs the headers in the `freetype2/` directory